bucket/neovim-gtk.json

```
@@ -0,0 +1,28 @@
+{
+    "homepage": "https://github.com/daa84/neovim-gtk",
+    "version": "0.1.1_545",
+    "description": "GTK ui for neovim written in rust using gtk-rs bindings. With ligatures support.",
```

```suggestion
    "description": "A highly configurable text editor for efficiently creating and changing any kind of text. (Neovim/GTK version, with ligatures support)",
```
Let's re-use the description for vim, OK?